packages/terra-dropdown-button/tests/jest/DropdownList.test.jsx

```diff
@@ -89,4 +115,58 @@ describe('Dropdown List', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should set the aria-label property to empty on keydown on Mac', () => {
+    // Set the mock implementation for isMac
+    SharedUtil.isMac.mockReturnValue(true);
+
+    const wrapper = shallowWithIntl(
+      <IntlProvider locale="en" messages={translationsFile}>
+        <DropdownList id="dropdownList" requestClose={() => { }}>
+          <Item id="firstItem" label="1st Option" onSelect={() => { }} />
+          <Item label="2nd Option" onSelect={() => { }} />
+          <Item label="3rd Option" onSelect={() => { }} />
+        </DropdownList>
+      </IntlProvider>,
+    ).dive().dive();
+
+    wrapper.instance().listRef = listRefMock;
+    const firstListItem = wrapper.find('#firstItem');
+    const firstListItemAriaLabelValue = firstListItem.props()['aria-label'];
+    const expectedAriaLabelValueInitial = `${translationsFile['Terra.dropdownButton.expanded']}1st Option,(1 of 3)`;
+    expect(firstListItemAriaLabelValue).toEqual(expectedAriaLabelValueInitial);
+
+    // Simulate keydown event
+    wrapper.instance().handleKeyDown(eventMock);
+    const updatedFirstListItemAriaLabelValue = wrapper.find('#firstItem').props()['aria-label'];
+    const expectedAriaLabelValue = '1st Option,(1 of 3)';
+    expect(updatedFirstListItemAriaLabelValue).toEqual(expectedAriaLabelValue);
```

Comment:
Let's put all the expect statements at the bottom for better readability.
```suggestion

    // Simulate keydown event
    wrapper.instance().handleKeyDown(eventMock);
    const updatedFirstListItemAriaLabelValue = wrapper.find('#firstItem').props()['aria-label'];
    const expectedAriaLabelValue = '1st Option,(1 of 3)';
    
    expect(firstListItemAriaLabelValue).toEqual(expectedAriaLabelValueInitial);
    expect(updatedFirstListItemAriaLabelValue).toEqual(expectedAriaLabelValue);
```

Reply:
Implemented as suggested.